.gitignore

```diff
@@ -1,15 +1,134 @@
-*.iml
+# Created by https://www.toptal.com/developers/gitignore/api/androidstudio
```

Comment:
This .gitignore file was created via gitignore.io which adds more files which the default .gitignore doesn't include. I noticed there were some .idea files included in the repository which are user specific files and unnecessary to have.

Reply:
Good.